logging/src/main/java/tech/pegasys/teku/logging/SubCommandLogger.java

```diff
@@ -30,6 +30,10 @@ public void display(final String message) {
     System.out.println(message);
   }
 
+  public void noOp(final String message) {
```

Comment:
Does there need to be a noOp for error messages as well? Instead of noop method on the SubCommandLogger thinking that NoOp SubCommandLogger that doesn't output would be nicer.

Reply:
removed the method.